### PR TITLE
Feature/ Infinite Loading for Data Viewer

### LINKED ISSUE #75: Infinite scrolling in data table
We should use an "infinite scrolling" behaviour in the data table (e.g. when showing experiment intersections) because it would take too long in specific scenarios to skip to a specific record.

### LINKED ISSUE #98: Make Chips Non Clickable
On the binary metrics page the chips who present values of the confusion matrix are clickable. But there is no related functionality. Thus this UI aspect is confusing for the user.

### PR DESCRIPTION
**Pull request by florian-papsdorf**
Until now, we preload all tuples for the data viewer. Now we want to load only a sequence of tuples when they are needed.

Closes #98 
Closes #75

### CODE REVIEW

**Comment by phpfs** [comment]
Sadly, this implementation seems to crash in case "rowGetter" is called with an index which was not yet populated. I attempted to at least fix this behavior by returning `{}` in this case, but thereby caused the table entries to be empty (until a scroll event occurs). Any thoughts on this? @florian-papsdorf

File: app/src/components/DataViewer/DataViewer.View.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+              width={width}
+              height={height}
+              rowClassName="table-row"
+              headerHeight={20}
+              rowHeight={30}
+              rowCount={tuplesCount}
+              rowGetter={({ index }: { index: number }): unknown => {
+                const dataRow: Record<string, string> = {};
+                // Todo: Evaluate whether this is a proper fix
+                if (data.data[index] === undefined) return dataRow;
```
Comment (phpfs) [inline app/src/components/DataViewer/DataViewer.View.tsx:36]: Find a proper fix for this please :)

Comment (martingraf4) [reply app/src/components/DataViewer/DataViewer.View.tsx]: done

**Comment by phpfs** [comment]
@martingraf4 @florian-papsdorf  The dataset/experiment previewer is not usable for more than 5 columns. Specifying a minWidth for all columns fixes the issue at least to a certain degree, but is still suboptimal. The react-virtualized table component seems to unsuitable since it does not allow horizontal scroll ("Table content can scroll vertically but it is not meant to scroll horizontally.", [source](https://github.com/bvaughn/react-virtualized/blob/master/docs/Table.md)) Can we use something else? :)

**Comment by phpfs** [comment]
Bug: When switching tabs on the binarymetrics page, the current scroll is not reset to zero.

**Comment by martingraf4** [comment]
we could also give the use the possibilty to remove columns

**Pull request merged**